.github/workflows/tests.yml.j2

```diff
@@ -9,6 +9,12 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+{% set branch = branch_name %}
```

Comment:
I think some doc strings about what this variables mean and what is the expected formatting of the values would help with maintenance going forward. :)